policyengine_canada/parameters/gov/provinces/mb/benefits/mbcb/reduction/four_children.yaml

```diff
@@ -0,0 +1,25 @@
+description: Manitoba Child Benefit phase out rate for families with four children.
```

Comment:
Manitoba phases the child benefit out for families with four children at this rate.